### PR TITLE
Update minor release acceptance criteria

### PR DESCRIPTION
**Pull request by atalman**
Update minor release acceptance criteria

### CODE REVIEW

**Review by albanD — APPROVED** [review]

File: RELEASE.md [hunk 1]
```diff
@@ -241,13 +241,11 @@ For patch releases issue tracker needs to be created. For patch release, we requ
 * https://github.com/pytorch/pytorch/issues/51886
 
 Only following issues are accepted:
-1. Fixes to regressions against previous major version
+1. Fixes to regressions against previous major version (e.g. regressions introduced in 1.13.0 from 1.12.0 are pickable for 1.13.1)
```
Comment (weiwangmeta) [inline RELEASE.md:244]: Should people interpret "from 1.12.0" == "after 1.12.0"?

**Review by weiwangmeta — APPROVED** [review]
👍

File: RELEASE.md (hunk 1 above)
Comment (atalman) [reply RELEASE.md:244]: I think its quite clear 1.12.0 - no regression
1.13.0 - regression is introduced

**Pull request closed without merging**